featurebyte/service/catalog.py

```diff
@@ -85,6 +88,32 @@ async def update_online_store(
             update_document_class=None,
         )
 
+    async def get_update_online_store_task_payload(
+        self,
+        document_id: ObjectId,
+        data: CatalogOnlineStoreUpdate,
+    ) -> CatalogOnlineStoreInitializeTaskPayload:
+        """
+        Get OnlineStoreInitializeTask payload
+
+        Parameters
+        ----------
+        document_id: ObjectId
+            Catalog id
+        data: CatalogOnlineStoreUpdate
+            Online store update payload
+
+        Returns
+        -------
+        CatalogOnlineStoreInitializeTaskPayload
+        """
+        document = await self.get_document(document_id=document_id)
```

Comment:
Should we check that that `document.online_store_id` is different from `data.online_store_id` here to avoid launching an unnecessary task?

Reply:
Sure, will add this check.